db/controllers/topic.controllers.js

```diff
@@ -0,0 +1,11 @@
+const { fetchTopics } = require("../models/topic.models");
+
+exports.getTopics = (req, res, next) => {
+  return fetchTopics()
+    .then((topics) => {
+      res.status(200).send(topics);
```

Comment:
this needs to be sent back on a meaningful key ie just sending it back as {topics}